{
	"identifier": "CVE-2020-8315",
	"description": "In Python (CPython) 3.6 through 3.6.10, 3.7 through 3.7.6, and 3.8 through 3.8.1, an insecure dependency load upon launch on Windows 7 may result in an attacker's copy of api-ms-win-core-path-l1-1-0.dll being loaded and used instead of the system's copy. Windows 8 and later are unaffected.",
	"type": "network",
	"severity": "medium",
	"state": "published",
	"datetime": "2020-01-28T17:15:00Z",
	"is_edited": false,
	"hardware": [],
	"software": [
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.13"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.12"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.11"
		},
		{
			"name": "python3",
			"version": "\u003c 3.7.7-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.10"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.17"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.16"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.15"
		},
		{
			"name": "python3",
			"version": "\u003c 3.8.2-r0",
			"architecture": "any",
			"manager": "apk",
			"vendor": "alpinelinux-3.14"
		},
		{
			"name": "python2.7",
			"version": "\u003c 2.7.18-8",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bullseye"
		},
		{
			"name": "python2.7",
			"version": "\u003c 2.7.16-2+deb10u1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-buster"
		},
		{
			"name": "python3.7",
			"version": "\u003c 3.7.3-2+deb10u3",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-buster"
		}
	],
	"products": [
		{
			"name": "python",
			"version": "\u003c= 3.6.10",
			"architecture": "any",
			"vendor": "python",
			"type": "software",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "python",
			"version": "\u003c= 3.7.6",
			"architecture": "any",
			"vendor": "python",
			"type": "software",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "python",
			"version": "\u003c= 3.8.1",
			"architecture": "any",
			"vendor": "python",
			"type": "software",
			"is_edited": false,
			"aliases": []
		}
	],
	"references": [
		"https://bugs.python.org/issue39401",
		"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8315"
	],
	"weaknesses": [
		"CWE-427"
	]
}